act-rules-format.bs

```diff
@@ -451,63 +496,67 @@ Definitions {#definitions}
   <dt><dfn>Accessibility requirement</dfn><dt>
   <dd>
     <p>A requirement is a condition that has to be satisfied in order to conform to a standard, or to comply with a contract, policy or regulation. An accessibility requirement is a requirement aimed at improving access for people with disabilities to an ICT product.</p>
-    <p>A common example of accessibility requirements are the WCAG 2.1 success criteria. There are other standards, including W3C standards that include recommendations for accessibility, such as WAI-ARIA and HTML. Accessibility requirements are also often found in company policies, regional standards or in legislation.</p>
+    <p>A common example of accessibility requirements are the WCAG 2.1 success criteria. There are other standards, including W3C standards that have recommendations for accessibility, such as WAI-ARIA and HTML. Accessibility requirements are also often found in company policies, regional standards or in legislation.</p>
   </dd>
 
   <dt><dfn>Accessibility requirements document</dfn><dt>
   <dd>
-    <p>A document, such as a standard, contract, policy or regulation, that includes [=accessibility requiremenst=]. For example, WCAG 2.1, WAI-ARIA 1.1, HTML 5.2, BBC HTML Accessibility Standards v2.0</p>
+    <p>A document, such as a standard, contract, policy or regulation, that includes [=accessibility requirements=]. For example, WCAG 2.1, WAI-ARIA 1.1, HTML 5.2, BBC HTML Accessibility Standards v2.0</p>
+  </dd>
+
+  <dt><dfn>Outcome</dfn></dt>
+  <dd>
+    <p>One of three types of conclusions that come from evaluating an ACT Rule on a [=test subject=]. An outcome can be:<p>
+    <ul>
+      <li>**Inapplicable:** No part of the test subject matches the applicability</li>
+      <li>**Passed:** The [=test target=] meets all expectations</li>
+      <li>**Failed:** The [=test target=] does not meet all expectations</li>
+    </ul>
+    <div class="note">
+      <p>**Note:** While *inapplicable* is a valid outcome for ACT Rules, it might not be a valid result for all [=accessibility requirements=]. Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to satisfied (passed and inapplicable) or not satisfied (failed).</p>
```

Comment:
**Editorial**: We've explained this in the accessibility mapping already. I think we can remove it here. The other 2 notes are more important.